firecloud/fiss.py

```diff
@@ -2521,10 +2540,12 @@ def main(argv=None):
                     'If no entity Type+Name is given, workspace-level ' +
                     'attributes will be listed.')
     # FIXME: this should explain that default entity is workspace
-    subp.add_argument('-e', '--entity', help="Entity name")
+    subp.add_argument('-e', '--entity', help="Entity name. For referenceData: hg38 or b37")
```

Comment:
I'm not sure this makes sense - what happens when the next reference comes out, or the user adds a custom reference (possibly non-human). This implies that b37 and hg38 are the only possibilities for `referenceData`.

Reply:
This is true - there could be newer references added to the UI (whereas now the only two available are hg38 and b37) so I will work out a way to make that apparent without being hardcoded.

If a user wants to add a custom reference, it will go into the Workspace Data tab and is not automatically prefixed with `referenceData_` as is the case with the Reference Data section which only offers hg38 and b37 so any custom references would be returned as a "workspace" attribute rather than a "reference" attribute. 
Since the user can add in any key value for custom references, we wouldn't be able to pattern match on anything definitive and return as "reference" data.